### PR TITLE
feat(toaster): add more options and update stories

### PR DESCRIPTION
**Pull request by matthprost**
## Type

- Enhancement

#### What is expected?

I added more option to toaster container, such as `className`, `position`, `newestOnTop`, `limit` and `id`.
I also updated stories to match our new way of doing.

### CODE REVIEW

File: src/components/Toaster/index.tsx [hunk 1]
```diff
@@ -122,6 +149,11 @@ const ToastContainer = (): JSX.Element => {
             toastClassName={localCss(styles.toast(theme))}
             autoClose={AUTOCLOSE_DELAY}
             icon={false}
+            newestOnTop={newestOnTop}
+            limit={limit}
+            className={className}
+            position={position}
+            containerId={id}
```
Comment (chambo-e) [inline src/components/Toaster/index.tsx:156]: should be sent by toast rather ?

File: src/components/Toaster/index.tsx [hunk 2]
```diff
@@ -122,6 +149,11 @@ const ToastContainer = (): JSX.Element => {
             toastClassName={localCss(styles.toast(theme))}
             autoClose={AUTOCLOSE_DELAY}
             icon={false}
+            newestOnTop={newestOnTop}
+            limit={limit}
+            className={className}
```
Comment (chambo-e) [inline src/components/Toaster/index.tsx:154]: Why ? 🤔

File: src/components/Toaster/index.tsx (hunk 1 above)
Comment (matthprost) [reply src/components/Toaster/index.tsx:156]: @chambo-e it's the container that manage all those settings

File: src/components/Toaster/index.tsx (hunk 2 above)
Comment (matthprost) [reply src/components/Toaster/index.tsx:154]: @chambo-e why for which one? All of them? I added some options for customizing a little more this component plus we need to allow `className` for each components.

File: src/components/Toaster/index.tsx (hunk 1 above)
Comment (chambo-e) [reply src/components/Toaster/index.tsx:156]: ContainerId is only there if we want multiple Container (which I assume we don't :))
https://fkhadra.github.io/react-toastify/multi-containers

If we want uniq toast we need to use toastId on the toast itself https://fkhadra.github.io/react-toastify/use-a-custom-id

File: src/components/Toaster/index.tsx (hunk 2 above)
Comment (chambo-e) [reply src/components/Toaster/index.tsx:154]: className yeah, why would we need it ?

**Review by QuiiBz — APPROVED** [review]

**Pull request merged**